test/end-to-end.js

```diff
@@ -426,6 +426,30 @@ describe('end-to-end', function() {
           done();
         });
     });
+
+    it('hides endpoint GET', function(done) {
```

Comment:
I think "hides" may be a bit misleading, since there is no GET endpoint at all. Perhaps this name would be better?

```js
it('comes without built-in GET endpoint')
```